docs/manual/manual.bbl

```diff
@@ -34,8 +34,8 @@ Joe Darcy.
 Werner Dietl, Stephanie Dietzel, Michael~D. Ernst, K{\i}van{\c{c}} Mu{\c{s}}lu,
   and Todd Schiller.
 \newblock Building and using pluggable type-checkers.
-\newblock In {\em ICSE: Proceedings of the 33rd International Conference on
-  Software Engineering}, pages 681--690, Waikiki, Hawaii, USA, May 2011.
+\newblock In {\em ICSE 2011, Proceedings of the 33rd International Conference
```

Comment:
The changes in this file are unrelated to the main changes in your pull request.  Please make a pull request contain only one set of changes.